### ISSUE TITLE
Wassa - UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 2280: character maps to <undefined>

### ISSUE THREAD

**Issue by Greg-Tarr** [flairNLP/flair#1765]
**Describe the bug**
It is a UnicodeDecodeError that occurs when using the flair.datasets.WASSA datasets.

Can be seen here:

> λ python wassa-sad.py
2020-07-17 23:49:59,174 http://saifmohammad.com/WebDocs/EmoInt%20Train%20Data/sadness-ratings-0to1.train.txt not found in cache, downloading to C:\Users\me\AppData\Local\Temp\tmpqf9ay00c
92977B [00:00, 211589.48B/s]
2020-07-17 23:49:59,950 copying C:\Users\me\AppData\Local\Temp\tmpqf9ay00c to cache at C:\Users\me\.flair\datasets\wassa_sadness\sadness-ratings-0to1.train.txt
2020-07-17 23:49:59,960 removing temp file C:\Users\me\AppData\Local\Temp\tmpqf9ay00c
Traceback (most recent call last):
  File "wassa-sadness2-pooled.py", line 15, in <module>
    corpus = WASSA_SADNESS()
  File "D:\Programs\Python\Python37\lib\site-packages\flair\datasets\document_classification.py", line 1607, in __init__
    _download_wassa_if_not_there("sadness", data_folder, dataset_name)
  File "D:\Programs\Python\Python37\lib\site-packages\flair\datasets\document_classification.py", line 1466, in _download_wassa_if_not_there
    next(f)
  File "D:\Programs\Python\Python37\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 2280: character maps to <undefined>

**To Reproduce**
`from flair.datasets import WASSA_SADNESS`
`corpus = WASSA_SADNESS()`

**Expected behavior**
The above error will occur due to the encoding left unspecified on lines 1464, 1465 in flair.datasets.document_classification.py

**Environment (please complete the following information):**
 - OS [e.g. iOS, Linux]: Windows 10 - Python 3.7.4
 - Version [e.g. flair-0.3.2]: 0.5.1 - but appears in non-master branches

**Additional context**
I changed the lines 1464, 1465 in flair.datasets.document_classification.py to include the encoding when opening files and it has solved the problem:

```
1464:   with open(path, "r", encoding="UTF-8") as f:
1465:       with open(data_file, "w", encoding="UTF-8") as out:
```

**Comment by marcelmmm** [comment]
Had the same error. After the encoding was specified it worked.

**Comment by marcelmmm** [comment]
👍

**Comment by alanakbik** [comment]
Closed by #1766 - thanks @Greg-Tarr !
